components/UI/CopyToClipboard.js

```diff
@@ -12,8 +13,7 @@ const CopyToClipboard = ({ content, title }) => {
     const copyToClipboard = async () => {
         navigator.clipboard.writeText(content);
         setWasContentCopied(true);
-        await new Promise(r => setTimeout(r, 2000));
-        setWasContentCopied(false);
+        await waitFor(() => setWasContentCopied(false));
```

Comment:
Actually, when I look at all the use cases of `waitFor()` maybe I do want to get back to my comment about `_.delay` -- why do we actually `await` here in this case?

What I'm saying is: for all use cases here I think we can just use `_.delay`. I'll give two examples:

**First example**

This here would just be:
```js
_.delay(() => setWasContentCopied(false), 2500);
```

No need for `await` as far as I can tell.

Which also means we can remove the `async` from `const copyToClipboard = async () => {`.